### PR TITLE
Waypoint Pathfinding

### PR DESCRIPTION
**Pull request by RexBerry**
When moving to a waypoint, find the shortest path that does not exit the flight boundary.

### CODE REVIEW

File: flight/waypoint/graph.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        Parameters
+        ----------
+        node : GraphNode[ValueT, WeightT]
+            The other node.
+
+        Returns
+        ------
+        bool
+            True if the nodes were disconnected, False if they weren't
+            connected before.
```
Comment (cjhr95) [inline flight/waypoint/graph.py:80]: Docstring needs an Exceptions section for the KeyError

**Review by cjhr95 — CHANGES_REQUESTED** [review]
Docstring fix

File: flight/waypoint/graph.py (hunk 1 above)
Comment (RexBerry) [reply flight/waypoint/graph.py:80]: Is that necessary if the function catches the exception?

**Review by cjhr95 — APPROVED** [review]
Fine, I guess so.

**Pull request merged**